module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/RecordDataFormatterTest.php

```diff
@@ -110,7 +110,8 @@ protected function getViewHelpers($container): array
             'record' => $record,
             'recordLinker' => new \VuFind\View\Helper\Root\RecordLinker($this->getMockRecordRouter()),
             'schemaOrg' => new \VuFind\View\Helper\Root\SchemaOrg(
-                new \Laminas\View\Helper\HtmlAttributes()
+                new \Laminas\View\Helper\HtmlAttributes(),
+                $this->getMockBuilder(\VuFind\View\Helper\Root\Ils::class)->disableOriginalConstructor()->getMock()
```

Comment:
You can use `$this->createMock(\VuFind\View\Helper\Root\Ils::class)` to be more concise while achieving the same result as using the mock builder.